README.md

```diff
@@ -227,7 +227,11 @@ Future work and Limitations
 Change Log (From version 2.2.0 and onwards)
 ==============
 
-### __2.6.0-SNAPSHOT__
+### __2.6.1-SNAPSHOT__
```

Comment:
Minor version should be bumped since we added functionality -> 2.7.0
Also remember to bump the version in the pom.xml file to 2.7.0